src/library-authoring/list-libraries/specs/LibraryListPage.spec.jsx

```diff
@@ -92,4 +135,86 @@ describe('list-libraries/LibraryListPage.jsx', () => {
     expect(mockLibraryFetcher).toHaveBeenNthCalledWith(4, { params: { ...commonParams, page: 2 } });
     expect(mockLibraryFetcher).toHaveBeenNthCalledWith(5, { params: { ...commonParams, page: 1 } });
   });
+
+  it('shows empty page for empty library list', () => {
+    const newProps = update(props, {
+      libraries: { count: { $set: 0 } },
+    });
+
+    const container = ctxMount(
+      <BrowserRouter>
+        <InjectedLibraryListPage {...newProps} />
+      </BrowserRouter>,
+    );
+
+    expect(container.find('.library-list .library-item').length).toBe(0);
+    expect(container.find('.empty-sheet-wrapper').exists()).toBeTruthy();
+
+    const emptyHeadingText = container.find('.empty-heading-text').text();
+    expect(emptyHeadingText).toEqual('Add your first library to get started');
+  });
+
+  it('shows the create form when clicking the new library button on a empty page', () => {
+    const container = ctxMount(
+      <BrowserRouter>
+        <Router history={history}>
+          <InjectedLibraryListPage {...props} />
+        </Router>
+      </BrowserRouter>,
+    );
+
+    const emptyPage = container.find('.empty-sheet-wrapper').at(0);
```

Comment:
You also can remove `.at(0)` because you wouldn't get more then one element

Reply:
done